services/movie.py

```diff
@@ -0,0 +1,49 @@
+from db.models import Movie
+from typing import Any
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> Any:
+    # If genres_ids and actors_ids are not provided,
+    queryset = Movie.objects.all()
+
+    # If only genres_ids is provided
+    if genres_ids and actors_ids is None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    # If only actors_ids is provided
+    if genres_ids is None and actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    # If both genres_ids and actors_ids are provided
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
```

Comment:
You can simplify this and write only with 2 conditions:
1. genres_ids is not None
2. actors_ids is not None